airbyte-config/models/src/main/resources/types/StandardSourceDefinition.yaml

```diff
@@ -25,3 +25,10 @@ properties:
     type: string
   icon:
     type: string
+  sourceType:
```

Comment:
should this be required?

Reply:
Probably. This is in the OSS table, and we don't have preset values for custom connectors. So some entries won't have this.